control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -903,7 +903,6 @@ func (r *reconciler) reconcileCloudCredentialSecrets(ctx context.Context, hcp *h
 		}
 		for arn, secret := range map[string]*corev1.Secret{
 			hcp.Spec.Platform.AWS.RolesRef.IngressARN:       manifests.AWSIngressCloudCredsSecret(),
-			hcp.Spec.Platform.AWS.RolesRef.StorageARN:       manifests.AWSStorageCloudCredsSecret(),
 			hcp.Spec.Platform.AWS.RolesRef.ImageRegistryARN: manifests.AWSImageRegistryCloudCredsSecret(),
```

Comment:
can you add the logic to ensure the guest cluster secret is deleted?

Reply:
Is this also done with other secrets? Can you point me to an example?

Reply:
This is first time. GIven we are going from having the secret in guest cluster to having it in management cluster side.
It's to ensure existing clusters don't have the stale secret guest side.

Reply:
We (Storage Team) decided to implement this together with the clean up logic that will delete old operands from guest cluster. This will be done in another PR (CC @gnufied).